WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -154,7 +174,9 @@ open class QuickStartTourGuide: NSObject {
 
     private func addSiteMenuWayPointIfNeeded(for tour: QuickStartTour) -> QuickStartTour {
 
-        if currentTourOrigin == .blogDashboard && tour.shownInBlogDetails && !UIDevice.isPad() {
+        if currentEntryPoint == .blogDashboard &&
+            tour.possibleEntryPoints == [.blogDetails] &&
```

Comment:
[nit] What about adding a computed variable to `TourState` called `mustBeShownInBlogDetails` and then call it here and in `currentTourMustBeShownFromBlogDetails`?

Reply:
Good suggestion! Instead of adding it to `TourState,` I opted to add it to `QuickStartTour` and provide a default implementation 9f03e03